test/jdk/java/awt/FontMetrics/ExtremeFontSizeTest.java

```diff
@@ -24,16 +24,19 @@
 import java.awt.Font;
 import java.awt.FontMetrics;
 import java.awt.Graphics2D;
+import java.awt.GraphicsEnvironment;
 import java.awt.Rectangle;
 import java.awt.font.FontRenderContext;
 import java.awt.font.GlyphVector;
 import java.awt.geom.AffineTransform;
 import java.awt.geom.Rectangle2D;
 import java.awt.image.BufferedImage;
+import java.util.HashMap;
+import java.util.Map;
 
 /*
  * @test
- * @bug 8328896
+ * @bug 8328896 8357672
```

Comment:
I thought Freetype rasterizer is used on all platforms(Linux, Windows & macOS) by default. Since Freetype is part of JDK.

When i run this test on macOS without any updates from this PR on baseline code, i see that all Fontmetrics in System.out() are non-zero and match before and after extreme parameters. Is this behaviour okay?

I was expecting mismatch in FontMetrics on macOS also.
If this test is supposed to catch issues on particular platforms. Do we need to make it specific to some platforms?